app/uk/gov/hmrc/exports/repositories/SubmissionRepository.scala

```diff
@@ -70,10 +70,10 @@ class SubmissionRepository @Inject()(implicit mc: ReactiveMongoComponent, ec: Ex
     }
   }
 
-  def updateStatus(eori: String, convId: String, status: Option[String]): Future[Boolean] =
+  def updateStatus(eori: String, convId: String, mrn: String, status: Option[String]): Future[Boolean] =
     if (status.isDefined) {
       val finder = BSONDocument("eori" -> eori, "conversationId" -> convId)
-      val modifier = BSONDocument("$set" -> BSONDocument("status" -> status.get))
+      val modifier = BSONDocument("$set" -> BSONDocument("mrn" -> mrn, "status" -> status.get))
```

Comment:
Hmm, I think that for this case we shouldn't update `mrn` and `status` inside `updateStatus` method. This method should only update status. You can use diffrent one `updateSubmission`, for me it's not a good place to do it.

Reply:
I've renamed the method updateStatusAndMrn. Its only called in one place.
Scalafmt run as well. but changed loads of files I've not modified..